modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -179,6 +179,15 @@ ComputeMultipleInelasticStress::initialSetup()
     paramError("damage_model",
                "Damage Model " + _damage_model->name() +
                    " is not compatible with ComputeMultipleInelasticStress");
+
+  // This check prevents the hierarchy from silently skipping substepping without informing the user
+  for (unsigned int model_number = 0; model_number < _num_models; ++model_number)
```

Comment:
Just a _suggestion_ (automatically uses the correct type for the counter - requires `#include "libmesh/int_range.h"`).

```suggestion
  for (auto model_number: index_range(_models))
```

Reply:
I just _made_ that change.